### ISSUE TITLE
MS Word. Move [Abbrev] button to Zotero menu ribbon.

### ISSUE THREAD

**Issue by JohnLukeBentley** [Juris-M/zotero#45]
>>     ... that button would better live on the |Zotero| menu/ribbon.

@fbennett wrote, in [MS Word. Manually entered abbreviations aren't preserved between Word and JurisM restarts #43](https://github.com/Juris-M/zotero/issues/43#issuecomment-373908103).
> Yes. I'd be every happy to accept a pull request if someone stepped up to make that happen.

**Comment by fbennett** [comment]
Been thinking about this. I had been thinking about it in terms of setting an icon in the Zotero plugin ribbon. I wouldn't want to go there, because of the maintenance cost -- JM currently uses the Zotero integration plugins, unmodified. Any customization touching plugin code would trigger a need to cone and maintain a the full plugins architecture, which I don't have time to do.

But an Abbrevs tab could easily be added to the Doc Prefs menu. That would require two clicks to get to, but the document-level context would be clear, and it would allow the removal of the (ugly) Abbrevs button in the citation popups.

One to think about, anyway.

**Comment by JohnLukeBentley** [comment]
Yes it makes sense that you wouldn't want to change your copy of the Zotero plugin for the sake of a mere button, given a hefty maintenance cost. 

I'm not sure what you mean by the "Doc Prefs" Menu. The file menu? Anyway, yes, moving the button to its own menu could be a less expensive option.

However, I'm not sure if the usability cost of having the set of relevant commands split across two menus would outweigh the gain of the conceptual clarity of the scope of the abbrevs button. Its current location certainly is convenient in terms of mouse movement.

... but, looking at what the zotero plugin has done, it's done two things
* Created and added an entire menu "Zotero"; and, on that menu ...
* Added it's own set of buttons in a group.

I wonder if you might be able to, in a maintenance cheap manner, just create your own "JurisM" group, next to the Zotero group, on the Zotero menu. That is, that after you, at regular intervals, import that latest version of the Zotero plugin you run a script (or some such) to copy a folder over containing your code for the group and also add a line somewhere in the Zotero code to hook it all up. 

That is said in ignorance of how you are currently importing (and how you must be importing, for ease) the Zotero plugin and the degree to which the Zotero plugin is amenable to cheap modification.

It is, yes, one to think about.

**Comment by JohnLukeBentley** [comment]
@fbennett You might want to re-open this one too.
